measureme/src/profiling_data.rs

```diff
@@ -21,6 +29,15 @@ impl ProfilingData {
             fs::read(paths.string_index_file).expect("couldn't read string_index file");
         let event_data = fs::read(paths.events_file).expect("couldn't read events file");
 
+        let event_data_format = read_file_header(&event_data, FILE_MAGIC_EVENT_STREAM)?;
+        if event_data_format != CURRENT_FILE_FORMAT_VERSION {
```

Comment:
Do we need to support `CURRENT_FILE_FORMAT_VERSION - 1` for perf.rlo?

cc @Mark-Simulacrum

Reply:
I imagine that we don't care too much in practice, we just want me to be aware when we make changes -- as it currently stands, we don't really support this too well, but we can either just deal with commits failing or some other way to migrate to a new version. Once we're including measureme in the artifacts rustc CI produces that would be helpful as well.

Reply:
Ok. We'll be sure to coordinate with you when we bump rustc's dependency version to the one with breaking changes.